dataland-frontend/src/components/forms/CreateEuTaxonomyForNonFinancials.vue

```diff
@@ -374,7 +376,7 @@ export default defineComponent({
     postEuTaxonomyDataForNonFinancialsResponse: null as AxiosResponse<DataMetaInformation> | null,
     message: "",
     namesOfAllCompanyReportsForTheDataset: [] as string[],
-    templateDataset: undefined as undefined | EuTaxonomyDataForNonFinancials,
+    templateDataset: {} as EuTaxonomyDataForNonFinancials,
```

Comment:
Why can't this be initalized as `undefined` ?
The underlying child "UploadReports" has already a default value "{}" in place in case that "templateDataset.referencedReports" is undefined.

Reply:
Ok I was just interested in the thought behind it.
Then it is fine. Closing this.